rclrs/src/node.rs

```diff
@@ -71,7 +71,7 @@ pub struct Node {
     pub(crate) guard_conditions_mtx: Mutex<Vec<Weak<GuardCondition>>>,
     pub(crate) services_mtx: Mutex<Vec<Weak<dyn ServiceBase>>>,
     pub(crate) subscriptions_mtx: Mutex<Vec<Weak<dyn SubscriptionBase>>>,
-    _parameter_map: ParameterOverrideMap,
+    _parameter: ParameterInterface,
```

Comment:
I like this pattern from `rclcpp`, we should do the same for services, subscriptions, etc.